include/cuco/static_multimap.cuh

```diff
@@ -666,6 +666,40 @@ class static_multimap {
   template <typename InputIt>
   size_type count(InputIt first, InputIt last, cuda::stream_ref stream = {}) const;
 
+  /**
+   * @brief Retrieves the matched key-value pair in the multimap corresponding to all probe keys in
+   * the range
+   * `[first, last)`
+   *
+   * If key `k = *(first + i)` has a match `m` in the multimap, copies a `cuco::pair{k, m}` to
+   * unspecified locations in `[output_begin, output_end)`. Else, does nothing.
+   *
+   * @note This function synchronizes the given stream.
+   * @note Behavior is undefined if the size of the output range exceeds
+   * `std::distance(output_begin, output_end)`.
+   * @note Behavior is undefined if the given key has multiple matches in the set.
```

Comment:
Undefined or rather "will only retrieve the first occurrence found"?

Reply:
The undefined part is that if multiple matches exist in the map, the "first" occurrence is not deterministic.